.github/workflows/agent.yaml

```diff
@@ -65,6 +65,16 @@ jobs:
           fetch-depth: 0
           token: ${{ steps.app-token.outputs.token }}
 
+      - name: Setup uv
+        # yamllint disable-line rule:line-length rule:comments
+        uses: astral-sh/setup-uv@681c641aba71e4a1c380be3ab5e12ad51f415867  # v7.1.6
+        with:
+          enable-cache: true
```

Comment:
@dobbyphus, i dont' think we need to enable the cache explicitly, on github it will be enabled by default.